lib/repl.js

```diff
@@ -416,7 +418,9 @@ function REPLServer(prompt,
       e.stack = e.stack.replace(/(\s+at\s+repl:)(\d+)/,
                                 (_, pre, line) => pre + (line - 1));
     }
-    if (isError && e.stack) {
+    if (process.listenerCount('uncaughtException') !== 0) {
+      process.emit('uncaughtException', e);
+    } else if (isError && e.stack) {
```

Comment:
I don’t think we want the event being emitted and information being written to stderr to be mutually exclusive? I’d generally expect that the REPL shows global errors that aren’t caught, even if there is an `uncaughtException` listener…